### ISSUE TITLE
Rare edge case where the answer was sort of hidden.

### ISSUE THREAD

**Issue by lordmatt** [pfefferle/wordpress-webmention#467]
I spent all morning trying to work out why WebMention installs on more than one blog all told me 403 - Forbidden for one other blog. It turned out that the source sight was giving the target sites a 500 when they came to check for a WebMention link.

I imagine this is really rare and could only happen to me. I'm not even sure there is anything that needs to be done with this issue. But, for completeness, and the fact I ended up asking Mr Pfefferle for help (and like a star, help was offered), I'm noting this happened.

Perhaps, if it is trivial to add, some message about the other site misbehaving might be warranted. If not, maybe just close this ticket on the grounds that I am sometimes a nitwit.

TL;DR: "The source site is misbehaving while being verified" is a thing that can happen.

**Comment by edent** [comment]
I'm also getting this. Here's an example - https://boingboing.net/2025/07/25/developer-recreates-creepy-cold-war-numbers-stations-using-just-1kb-of-code.html

Results in

<img width="1528" height="288" alt="Image" src="https://github.com/user-attachments/assets/037e2796-f44f-4efd-b445-868f904f0cbf" />

Looking at the output of `wp_remote_get()`, it is hitting the CloudFlare CAPTCHA.  I don't think there's much can be done about that :-/
